submitted_models/coro_crystal_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,907 @@
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+
+      <!-- Physical robot has B0247 with LN008 with a 180 deg hor x 130 deg vert FOV
+           Since fisheye cameras are not modeled, two cameras are used in the model -->
+      <sensor name="camera_left" type="camera">
+        <pose>0.098 -0.002 0.03 0 0 0.785398163</pose>
+        <always_on>1</always_on>
+        <update_rate>25</update_rate>
+        <camera name="camera_left">
+          <horizontal_fov>1.658</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+              <fx>329.9</fx>
+              <fy>329.9</fy>
+              <!-- cx = ( width - 1 ) / 2 -->
+              <cx>359.5</cx>
+              <!-- cy = ( height - 1 ) / 2 -->
+              <cy>539.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+          <image>
+            <width>720</width>
+            <height>1080</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.01</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+      </sensor>
+
+      <sensor name="camera_right" type="camera">
+        <pose>0.098 0.002 0.03 0 0 -0.785398163</pose>
+        <always_on>1</always_on>
+        <update_rate>25</update_rate>
+        <camera name="camera_right">
+          <horizontal_fov>1.658</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+              <fx>329.9</fx>
+              <fy>329.9</fy>
+              <!-- cx = ( width - 1 ) / 2 -->
+              <cx>359.5</cx>
+              <!-- cy = ( height - 1 ) / 2 -->
+              <cy>539.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+          <image>
+            <width>720</width>
+            <height>1080</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.01</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+      </sensor>
+      
+      <sensor name="bottom_laser" type="gpu_ray">
+        <pose>0.0 0 -0.14 0 -1.57 0</pose>
```

Comment:
this places the lidar below the ground plane when the UAV is at rest, see red cube in the image below. Move this sensor up?
![crystal_bottom_lidar](https://user-images.githubusercontent.com/4000684/115458942-499f0480-a1db-11eb-9566-b213083db6a7.png)

Reply:
I moved the sensor to the correct location.

Reply:
![crystal_bottom_lidar_updated](https://user-images.githubusercontent.com/4000684/115595529-e3be8580-a28b-11eb-9813-5fc61d0e26ee.png)

looks good to me, thanks.